routers/user/home.go

```diff
@@ -194,7 +199,19 @@ func Issues(ctx *context.Context) {
 		page = 1
 	}
 
-	repoID := ctx.QueryInt64("repo")
+	// Replace regexp return value with commas, and split
+	repoIDStrings := strings.Split(issueRepoIDsPattern.ReplaceAllString(ctx.Query("repos"), ","), ",")
+	var repoIDs []int64
+	for _, IDString := range repoIDStrings {
+		// Ensure nonempty string entries
+		if IDString != "" && IDString != "0" {
+			IDint64, err := strconv.ParseInt(IDString, 10, 64)
```

Comment:
local variable name capital letter should be lower.

Reply:
done